dev-python/python-eventlib/python-eventlib-0.2.1-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2015 Gentoo Foundation
```

Comment:
Why are you calling this a 'version bump' if you're not bumping version? This is really confusing.

Reply:
how should I call a change from 0.2.1 to 0.2.1-r1?

Reply:
Call it by what you're actually changing.

Reply:
I mean *meaningful* change in the ebuild. The user cares that you're changing the deps, now what revision the ebuild will have now.